docs/guides/configure-scan.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Configure the scan command
+---
+
+# Configure the scan to meet your needs
+
+Bearer offers a variety of ways to configure the core `scan` command to best meet your needs. Here are some common situations. For a full list of options, see the [commands reference](/reference/commands/). For many of the command flags listed below, you can also define them in your `bearer.yml` [config file](/reference/config/).
+
+## Select a report type
+
+There are a variety of [report types](/explanations/reports/) to choose from. Bearer defaults to the Security report, but you can select any other type with the `--report` flag.
+
+```bash
+bearer scan . --report privacy
+```
+
+## Select a scanner type
+
+Did you know that Bearer can also detect hard-coded secrets in your code? In addition to the default SAST scanner, there's a built-in secrets scanner. Use the `--scanner` flag to change [scanner types](/explanations/scanners/).
+
+```bash
+bearer scan . --scanner secrets
+```
+
+## Skip or ignore specific rules
+
+Sometimes you want to ignore one or more rules, either for the entire scan or for individual blocks of code. Rules are identified by their id, for example: `ruby_lang_exception`.
+
+### Skip rules for the entire scan
+
+To ignore rules for the entire scan you can use the `--skip-rule` flag with the `scan` command.
+
+Using `--skip-rule`:
+
+```bash
+# skip a single rule
+bearer scan . --skip-rule ruby_lang_exception
+
+# skip multiple rules
+bearer scan . --skip-rule ruby_lang_exception,ruby_lang_cookies
+```
+
+Using `bearer.yml`
+
+```yaml
+rule:
+  skip-rule: [ruby_lang_exception, ruby_lang_cookies]
```

Comment:
You can also use the format
skip-rule:
  - ruby_lang_exception
  - ruby_lang_cookies

Reply:
init uses the [ ] syntax. YAML pros can adapt as needed